src/test/integration/documentClient.spec.ts

```diff
@@ -1,14 +1,12 @@
 ﻿import * as assert from "assert";
 import { Constants, CosmosClient, IHeaders } from "../../";
-import testConfig from "./../common/_testConfig";
+import { endpoint, masterKey } from "./../common/_testConfig";
+
+const client = new CosmosClient({ endpoint, auth: { masterKey } });
```

Comment:
Remove unnecessary client

Reply:
This one is still used here. But this test is also testing `client.documentClient` directly so we should probably just delete it entirely...

Reply:
Ah yeah, documentClient leaking is still a hack we've done to expose the session management stuff until we refactor documentClient away.